Readme.md

```diff
@@ -82,7 +82,7 @@ let package = Package(
   targets: [
     Target(
```

Comment:
For a single target project, I think we can get even simpler with our recommended SPM install and do this:

```swift
import PackageDescription

let package = Package(
  name: "MyApp",
  dependencies: [
    .Package(url: "https://github.com/Moya/Moya", majorVersion: 8)
  ]
)
```

That should be all the majority of people need. You don't need `Target`s unless you have a multi-target package (maybe that will be more common when you can use SPM to build Apps). Also the dependencies on a `Target` level are usually used to specify inter-Target dependencies and not external package dependencies like Moya.